test/tests/mqtrigger/kafka/test_kafka.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+#test:disabled
+
+# Create a function and trigger it using Kafka
+# This test requires Kafka & MQ-Kafka component of Fission installed in the cluster
+set -euo pipefail
+set +x
+
+nodeenv="node-kafka"
+goenv="go-kafka"
+producerfunc="producer-func"
+consumerfunc="consumer-func"
+
+log() {
+    echo $1
+}
+export -f log
+
+test_mqmessage() {
+    echo "Checking for valid response"
+
+    while true; do
+      response0=$(kubectl -nfission logs -l=messagequeue=kafka)
+      echo $response0 | grep -i $1
+      if [[ $? -eq 0 ]]; then
+        break
+      fi
+      sleep 1
+    done
+}
+export -f test_mqmessage 
+
+waitBuild() {
+    log "Waiting for builder manager to finish the build"
+    
+    while true; do
+      kubectl --namespace default get packages $1 -o jsonpath='{.status.buildstatus}'|grep succeeded
```

Comment:
use `kubectl wait` instead of this custom wait/retry functionality

Reply:
Wait would mean we have to constantly watch the IO, would try implementing in another PR